test/viewmodels/checkAnswers/expenses/officeSupplies/OfficeSuppliesDisallowableAmountSummarySpec.scala

```diff
@@ -26,7 +26,9 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 
 class OfficeSuppliesDisallowableAmountSummarySpec extends AnyWordSpec with Matchers {
```

Comment:
these are extended in SpecBase already, think we should keep consistent with the scaffold and all other Specs